database/database.go

```diff
@@ -348,48 +350,15 @@ func (db *Database) SetDownloadInfo(downloadInfo downloader.DownloadInfo) (err e
 
 // GetServicesInfo returns services info.
 func (db *Database) GetServicesInfo() ([]imagemanager.ServiceInfo, error) {
-	allServiceVersions, err := db.getServicesFromQuery(`SELECT * FROM services`)
-	if err != nil {
-		return nil, err
-	}
-
-	maxVersionServices := make(map[string]imagemanager.ServiceInfo)
-
-	for _, service := range allServiceVersions {
-		if storedService, found := maxVersionServices[service.ServiceID]; found {
-			greater, err := semverutils.GreaterThan(service.Version, storedService.Version)
-			if err != nil {
-				return nil, aoserrors.Wrap(err)
-			}
-
-			if greater {
-				maxVersionServices[service.ServiceID] = service
-			}
-		} else {
-			maxVersionServices[service.ServiceID] = service
-		}
-	}
-
-	result := make([]imagemanager.ServiceInfo, 0)
-	for _, value := range maxVersionServices {
-		result = append(result, value)
-	}
-
-	return result, nil
+	return db.getServicesFromQuery(`SELECT * FROM services`)
```

Comment:
Previously function returned data for max service version only.  I guess it should return info for current service version. as I understood the one with state != cached

Reply:
It returns all service with all versions, filtering is done by `imagemanager` `GetServicesStatus` function.